scipy/special/add_newdocs.py

```diff
@@ -161,6 +161,61 @@ def add_newdoc(place, name, doc):
 
     """)
 
+
+add_newdoc("scipy.special", "agm",
+    """
+    agm(a, b)
+
+    Compute the arithmetic-geometric mean of `a` and `b`.
+
+    Start with a_0 = a and b_0 = b and iteratively compute::
+
+        a_{n+1} = (a_n + b_n)/2
+        b_{n+1} = sqrt(a_n*b_n)
+
+    until a_n = b_n.  The result is agm(a, b).
```

Comment:
Might instead say something like "`a_n` and `b_n` converge to the same limit as `n -> oo`; their common limit is `agm(a, b)`". (I guess it is more jargony, but OTOH one typically won't have `a_n = b_n` after finitely many steps.)